src/06-loops.js

```diff
@@ -117,7 +163,15 @@ function targetCount(array, target) {}
  *  firstIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function firstIndexFound(array, target) {}
+function firstIndexFound(array, target) {
+  let result = -1;
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === target) {
+      return (result = i);
+    }
+  }
+  return result;
+}
```

Comment:
You don't need the `result =` part in your return; `result = i` just evaluates to `i`, so you could simply write `return i`.